src/stories/SegmentedPicker/index.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
```

Comment:
remove import screen, if you want to use `screen.getByText`, you can get from  `const {getByText} = render(<SegmentedPicker {...defaultProps} />);`

Reply:
change:
const { getByText } = render(<SegmentedPicker {...defaultProps} />);

    segmentsArray.forEach((segment) => {
      expect(getByText(segment.label)).toBeInTheDocument();
    });